lib/Migration/Version010003Date20231208083112.php

```diff
@@ -122,6 +122,29 @@ public function changeSchema(IOutput $output, Closure $schemaClosure, array $opt
 			$table->addIndex(['image_gen_id'], 'assistant_i_stale_gens_id');
 		}
 
+		if (!$schema->hasTable('assistant_t_prompts')) {
```

Comment:
Small detail for this migration. We usually have a boolean flag named `$schemaChanged` which is initialized to false and becomes true if there actually are changes to the schema. At the end of `changeSchema()` we return the schema if there were changed or null if not.

Small optimization to avoid returning a schema when there were no changes.